frontends/php/include/test.inc.php

```diff
@@ -128,4 +137,124 @@ function add_to_testing_table($item) {
    $result = DBexecute($sql);
 
    return $result;
-}
\ No newline at end of file
+}
+
+/**
+ * Removes item from item_testing table 
+ *
+ * @param string $itemid   item to be removed
+ * 
+ * @return bool
+ */
+function remove_from_testing_table($itemid) {
+   $result = false;
+   $result = DB::delete('item_testing', array('itemid'=>$itemid));
```

Comment:
Not sure if I have latest DB class but it seems to me delete always returns true (or throws exception if database operation fails).

Reply:
Fixed with try catch.